connectors/email/src/main/java/io/camunda/connector/email/client/jakarta/utils/Header.java

```diff
@@ -4,6 +4,6 @@
  * See the License.txt file for more information. You may not use this file
  * except in compliance with the proprietary license.
  */
-package io.camunda.connector.email.client.jakarta;
+package io.camunda.connector.email.client.jakarta.utils;
```

Comment:
any reason to move all the models in `utils` ?

Reply:
Done

Reply:
Changed